spec/operations/beta/project_board_operations_spec.rb

```diff
@@ -13,8 +13,11 @@
 
       let(:user) { create(:user, :with_team) }
 
+      let(:story_operations_read_all_instance) { StoryOperations::ReadAll.new }
```

Comment:
```suggestion
      let(:story_operations_read_all_instance) { instance_double(StoryOperations::ReadAll, call: { active_stories: stories) }
```

I'm pretty sure this will work the way you want without having to use the `allow` for line 20. Can you test it and let me know if it worked?

Reply:
it worked! ty